routes/web.php

```diff
@@ -27,6 +27,7 @@
 
     Route::get('/print', 'PrintController@index')->name('print')->middleware('can:print.print');
     Route::post('/print/modify_balance', 'PrintController@modify_balance')->name('print.modify')->middleware('can:print.modify');;
+    Route::put('/print/print', 'PrintController@print')->name('print.print')->middleware('can:print.print');;
```

Comment:
This naming convention is so ugly. Can't we come up with something more meaningful?
What is the idiomatic way of doing this? @vendi95 

Probably worth to investigate it later.

Reply:
It might be worth it to refactor the module name to something that better represents printing than 'print'. I found 'printer' misleading, and 'printing' simply bad.